app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+
+  def create
+    user = Person.from_omniauth(request.env["omniauth.auth"])
+    session[:user_id] = user.id
+    redirect_to '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.